src/main/java/org/walkmod/javalang/compiler/actions/LoadStaticImportsAction.java

```diff
@@ -157,20 +158,19 @@ public void doPush(Symbol<?> symbol, SymbolTable table) throws Exception {
                         int modifiers = field.getModifiers();
                         if (Modifier.isStatic(modifiers)) {
 
-                            boolean isVisible = Modifier.isPublic(modifiers)
-                                    || (!Modifier.isPrivate(modifiers) && importPkgName.equals(pkgName));
-                            if (isVisible) {
-                                Class<?> type = field.getType();
-                                SymbolType st = new SymbolType(type);
-                                st.setParameterizedTypes(null);
-                                Symbol<?> s = new Symbol(field.getName(), st, n, ReferenceType.VARIABLE, true, null);
-                                table.pushSymbol(s);
-                            }
-                        }
-                    }
-                }
-            }
-        }
+							boolean isVisible = Modifier.isPublic(modifiers)
```

Comment:
This code follows a different format than you suggested to me. Could you format the code?